CHANGELOG.md

```diff
@@ -1,7 +1,17 @@
 # Changelog
 
+## 1.7.0-RC3
+
+- User Bulk Upload
+  - Error Handling
+
 ## 1.7.0-RC2
 
+- App Subscription
+  - Autosetup Process Worker implementation (adjust overlay)
+- Usermanagement
+  - User Invite IdP connection logic
+  
```

Comment:
@nidhigarg-bmw  please check once why new changeLog content was added in the RC-2 area. Where is this coming from? Any Idea?

Reply:
@jjeroch Not sure but I think these changes are done in dev. Where should I put these changelog now ?

Reply:
@nidhigarg-bmw please delete the 4 lines; they are incorrectly added. Please always have an eye on the ChangeLog and contact me in case such cases happen again, if we incorrectly update old changeLogs the history breaks and likely wrong information are added